### PR TITLE
FEATURE: ONE-4407 Manual resizing in URI Visualization

### PR DESCRIPTION
**Pull request by ivan-nejezchleb**
Apply manual resizing defs and grow to fit in URI Visualization based on FFs

---

[Check PR owner responsibilities](https://confluence.intgdc.com/display/Development/Code-reviews#Code-reviews-Ownerresponsibilities)

Supported PR commands:

Command | Description
--- | ---
`extended test - examples` | Live examples tests
`extended test - storybook` | Storybook screenshot tests

See more [options](https://confluence.intgdc.com/display/kbhr/How+to+work+with+git+and+Github#HowtoworkwithgitandGithub-Extendedchecks).

# PR Checklist

- [ ] Verify code changes ([Checklist](https://confluence.intgdc.com/display/Development/Code-reviews+checklist), [Best practices](https://confluence.intgdc.com/display/Development/Code-reviews+best+practices))
- [ ] [Verify pull-request formalities](https://confluence.intgdc.com/display/Development/Code-reviews)
- [ ] Change was tested by using [gdc-dev-release](https://confluence.intgdc.com/display/~tomas.vojtasek/Private+NPM) in [gdc-analytical-designer](https://github.com/gooddata/gdc-analytical-designer) and [gdc-dashboards](https://github.com/gooddata/gdc-dashboards) (if applicable)
- [ ] Migration guide (for major changes) is mentioned in [CHANGELOG.md](../blob/master/CHANGELOG.md).
- [ ] Successful `extended test - examples`
- [ ] Successful `extended test - storybook`
- [ ] Checked yarn.lock consistency (no dep. duplicities especially Goodstrap)

# Related PRs

- gdc-analytical-designer:
- gdc-dashboards:

# Related Jira tasks
- ONE-4407: https://jira.intgdc.com/browse/ONE-4407

### CODE REVIEW

**Comment by ivan-nejezchleb** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7637/ : SUCCESS in 4m 27s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/3754/ : SUCCESS in 4s

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/3999/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/3755/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/3360/ : UNSTABLE in 16m 51s

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7678/ : FAILURE in 47s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/3790/ : SUCCESS in 3s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7691/ : FAILURE in 48s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/3802/ : SUCCESS in 3s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7693/ : FAILURE in 6m 26s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/3803/ : SUCCESS in 3s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7694/ : SUCCESS in 6m 39s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/3804/ : SUCCESS in 4s

**Comment by ivan-nejezchleb** [comment]
extended test - storybook

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/4017/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/3812/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/3372/ : UNSTABLE in 11m 33s

**Comment by ivan-nejezchleb** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7700/ : SUCCESS in 3m 09s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/3814/ : SUCCESS in 3s

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/4019/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/3815/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/3374/ : UNSTABLE in 9m 48s

**Comment by ivan-nejezchleb** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7702/ : SUCCESS in 2m 45s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/3817/ : SUCCESS in 3s

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/4020/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/3818/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/3375/ : SUCCESS in 9m 50s

**Comment by gdgate** [comment]
Merge Failed.

Zuul merger could not merge this change into the base branch. This is most likely caused by merge conflicts. Please rebase the change and upload the rebased version. In case of further errors, contact the zuul administrator.

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by ivan-nejezchleb** [comment]
extended test - examples

**Comment by yenkins** [comment]
:arrow_right: Started Live Examples deploy to:
https://gdc-live-examples-1639.herokuapp.com

It will be destroyed on merge/close. See [ci-infra](https://github.com/gooddata/ci-infra/blob/master/jenkins/jobs/templates/orchestrators/client-pr-dispatcher.groovy) and [gdc-ci.yaml](https://github.com/gooddata/gooddata-react-components/blob/master/gdc-ci.yaml).

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7715/ : FAILURE in 3m 03s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/3833/ : SUCCESS in 7s

**Comment by gdgate** [comment]
Build in pipeline check-extended aborted.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7716/ : SUCCESS in 5m 27s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-oss-check/3835/ : SUCCESS in 4s

File: src/helpers/featureFlags.ts [hunk 1]
```diff
@@ -33,22 +33,34 @@ export function setConfigFromFeatureFlags(config: IChartConfig, featureFlags: IF
     return result;
 }
 
-export function getTableConfigFromFeatureFlags(featureFlags: IFeatureFlags): IPivotTableConfig {
+export function getTableConfigFromFeatureFlags(
+    config: IPivotTableConfig,
+    featureFlags: IFeatureFlags,
+    predicateEnvironment: boolean = true,
```
Comment (no23reason) [inline src/helpers/featureFlags.ts:39]: This parameter name is a bit confusing. What does it do? Why does it only affect `enableTableColumnsGrowToFit`?

Comment (ivan-nejezchleb) [reply src/helpers/featureFlags.ts:39]: renamed. GrowToFit is turned on only on Dashboards. In AD and URI Visualizations it is turned off without influence of FF

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/4026/ : SUCCESS in 12m 00s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-fossa-scanning-zuul-el7/89/ : SUCCESS in 6m 05s (non-voting)
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/7719/ : SUCCESS in 2m 51s
- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/1864/ : SUCCESS in 13s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/2769/ : SUCCESS in 16m 57s

**Pull request merged**